InfiniteLoopLMSSQL/tests/daos/AuthorDaoTest.java

```diff
@@ -0,0 +1,105 @@
+package daos;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Test;
+
+import com.st.lmssql.dao.GenericDao;
+import com.st.lmssql.daoImp.AuthorDaoImp;
+import com.st.lmssql.models.Author;
+import com.st.lmssql.utils.ConnectionFactory;
+
+public class AuthorDaoTest {
+
+	private static Connection con = ConnectionFactory.getMyConnection();

```

Comment:
It's generally not a good idea to run your tests against your production database ...